### DISCUSSION TITLE
Way to trigger journey based on count of particular event

### DISCUSSION THREAD

**Discussion by brijesh-ziroh** [parcelvoy/platform#477] [Q&A] [answered]
Hi,

I have requirement in which we need to trigger notification if we have received particular event more than some x times. Is this possible to achieve currently?

One way I can think of by creating one journey where on each event it will increment one counter at user level custom field & when the count is greater than x times then trigger the notification but handlebar by default don't support sum operation unless by registerting the custom helper function which can't be doable I guess. 

Let me know if there any other approach or workaround for this?

Thank you very much

**Comment by pushchris** [comment, marked as answer]
Hello @brijesh-ziroh! Your assumption on how to achieve what you are looking for is correct. We have a fully featured set of math operators you can use inside of Handlebar templates, you can find documentation on them here: https://docs.parcelvoy.com/how-to/campaigns/templates#numbers
